components/playlist/browser/resources/components/player.tsx

```diff
@@ -104,7 +104,7 @@ export default function Player () {
         videoElement.src !== currentItem?.mediaSource.url &&
         videoElement.src !== currentItem?.mediaPath.url
       ) {
-        videoElement.src = currentItem.mediaSource.url
+        videoElement.src = currentItem.mediaPath.url
```

Comment:
This is another regression - we should use mediaPath so that we can use cached file path if there is.